jf_agent/main.py

```diff
@@ -116,7 +119,7 @@ def main():
                 print(
                     f'Error connecting to Jira instance at {config.jira_url}, please validate your credentials. Error: {e}'
                 )
-            return
+            something_went_wrong = True
```

Comment:
Maybe a taste thing, but we *definitely* know we'll want to return `True` at this point, right?  So why not just `return True` instead of setting a variable?  The way you've written it, I have to wonder where else that variable might get set, or if it might get mutated before it's returned, or whatever else.